lua/lualine/themes/solarized_light.lua

```diff
@@ -0,0 +1,59 @@
+-- =============================================================================
+-- Filename: lua/lualine/themes/solarized_light.lua
+-- Author: shadman
+-- Credit: itchyny(lightline)
+-- License: MIT License
+-- =============================================================================
+
+local M = {}
+
+local colors = {
+  base3  =  '#002b36',
+  base2  =  '#073642',
+  base1  =  '#586e75',
+  base0  =  '#657b83',
+  base00   =  '#839496',
+  base01   =  '#93a1a1',
+  base02   =  '#eee8d5',
+  base03   =  '#fdf6e3',
+  yellow  =  '#b58900',
+  orange  =  '#cb4b16',
+  red     =  '#dc322f',
+  magenta =  '#d33682',
+  violet  =  '#6c71c4',
+  blue    =  '#268bd2',
+  cyan    =  '#2aa198',
+  green   =  '#859900',
+}
+if vim.o.background == 'light' then
+  colors.base03, colors.base3  =  colors.base3, colors.base03
+  colors.base02, colors.base2  =  colors.base2, colors.base02
+  colors.base01, colors.base1  =  colors.base1, colors.base01
+  colors.base00, colors.base0  =  colors.base0, colors.base00
+end
```

Comment:
in regards to this [comment](https://github.com/hoob3rt/lualine.nvim/pull/63#issuecomment-759161077)
> Can you tell me if solarized dark and light look different ? they look same on my device . even lightlines seem same . maybe something is overriding my configs.

if you remove this `if` block, the colors will be correct for the light theme. you already inverted the color definitions in `local colors = {...` 
eg:
- `light: base3  =  '#002b36', base03   =  '#fdf6e3',`
- ` dark:  base03  =  '#002b36', base3   =  '#fdf6e3',`

i've been using this branch w that modification and both themes are working correctly

Reply:
I had absulutely forgotten about that . Thanks for letting me know .

Reply:
👍  thanks for porting this over, it looks and works great!